source/NVDAObjects/window/excelChart.py

```diff
@@ -28,7 +28,7 @@
 import string
 import weakref
 import api
-
+from chartConstants import *
```

Comment:
I'd say it would be best to rename the constants file name so it starts with in underscore, otherwise people will think chart constants contain NVDA objects one can work with. Thanks.